app/models/evaluation_rubric.php

```diff
@@ -393,10 +393,22 @@ function rubricEvalScore($eventId, $conditions) {
         $sub = $evalSub->getEvalSubmissionsByEventId($eventId);
         $event = $this->Event->find('first', array('conditions' => array('Event.id' => $eventId)));
         $submitted = Set::extract('/EvaluationSubmission/submitter_id', $sub);
-
-        $rubricDetails = $this->find('all', array(
-            'conditions' => array_merge(array('evaluator' => $submitted), $conditions)
-        ));
+        
+        
+        $rubricDetails = array();
+        
+        //chunk find
+        $total = $this->find('count', array('conditions' => array_merge(array('evaluator' => $submitted), $conditions) ));
+        $limit = 100;
+        $pages = ceil($total / $limit);
+        for ($page = 1; $page < $pages + 1; $page++) {
```

Comment:
Though 100 limit seems small too me. DB query round trip is also expensive. With 1500 records, you will need 15 to 16 queries. Maybe increase it a little bit. 

Also another way to reduce memory usage is retrieve only needed fields for line 405.